### PR TITLE
Jormungandr: add info on base stoptimes

### PR DESCRIPTION
**Pull request by antoine-de**
in /disruptions, add info about the base stoptime:

``` json
impacted_objects": [
{
    "impacted_stops": 
[{
    "amended_arrival_time": "080224",
    "cause": "cow on tracks",
    "base_departure_time": "080100",  --> this field
    "base_arrival_time": "080100", --> this field
    "amended_departure_time": "080224"
},
    {
        "amended_arrival_time": "080400",
        "cause": "",
        "base_departure_time": "080242",
        "base_arrival_time": "080241",
        "amended_departure_time": "080400"
    }
],
"pt_object": 
{
    "embedded_type": "trip",
    "trip": {
                "id": "vjA"
            },
            "quality": ​0,
            "id": "vjA",
            "name": "vjA"
        }
    }
],
```

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/fields.py [hunk 1]
```diff
@@ -814,4 +814,4 @@ def get_all_impacts(_, val):
 
         utils.walk_protobuf(obj, get_all_impacts)
 
-        return [marshal(d, disruption_marshaller, display_null=False) for d in all_impacts.values()]
+        return [marshal(d, disruption_marshaller, display_null=False) for d in all_impacts.values()]
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py]: empty line

File: source/type/type.cpp [hunk 2]
```diff
@@ -502,15 +502,27 @@ void MetaVehicleJourney::cancel_vj(RTLevel level,
...
 
+const std::unique_ptr<VehicleJourney>& VehicleJourney::get_corresponding_base() const {
+    auto shifted_vj = get_base_canceled_validity_pattern();
+    for (const auto& vj: meta_vj->get_base_vj()) {
+        // if the validity pattern intersects
+        if ((shifted_vj.days & vj->base_validity_pattern()->days).any()) {
+            return vj;
+        }
+    }
+    static std::unique_ptr<VehicleJourney> none = nullptr;
```
Comment (xlqian) [inline source/type/type.cpp]: you may return a raw pointer of VJ then you can return nullptr here

**Comment by TeXitoi** [comment]
OK for me.

**Comment by xlqian** [comment]
Ok for me

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**